src/latexify/transformers/identifier_replacer_test.py

```diff
@@ -0,0 +1,99 @@
+"""Tests for latexify.transformer.identifier_replacer."""
+
+import ast
+
+import pytest
+
+from latexify import test_utils
+from latexify.transformers.identifier_replacer import IdentifierReplacer
+
+
+def test_invalid_mapping() -> None:
+    with pytest.raises(ValueError, match=r"'123' is not an identifier name."):
+        IdentifierReplacer({"123": "foo"})
+    with pytest.raises(ValueError, match=r"'456' is not an identifier name."):
+        IdentifierReplacer({"foo": "456"})
+
+
+def test_name_replaced() -> None:
+    source = ast.Name(id="foo", ctx=ast.Load())
+    expected = ast.Name(id="bar", ctx=ast.Load())
+    transformed = IdentifierReplacer({"foo": "bar"}).visit(source)
+    test_utils.assert_ast_equal(transformed, expected)
+
+
+def test_name_not_replaced() -> None:
+    source = ast.Name(id="foo", ctx=ast.Load())
+    expected = ast.Name(id="foo", ctx=ast.Load())
+    transformed = IdentifierReplacer({"fooo": "bar"}).visit(source)
```

Comment:
Can you also add a substr case? e.g. `transformed = IdentifierReplacer({"fo": "bar"}).visit(source)`

Reply:
SGTM, will do